TEST_APPS/device/socket_app/cmd_socket.cpp

```diff
@@ -150,25 +146,6 @@ class SInfo {
     {
         MBED_ASSERT(sock);
     }
-    SInfo(TCPServer *sock, bool delete_on_exit = true):
-        _id(id_count++),
-        _sock(sock),
-        _type(SInfo::TCP_SERVER),
```

Comment:
The enum(TCP_SERVER) should also be removed as it's not needed anymore.

Reply:
Done